pkg/api/v1beta1/types.go

```diff
@@ -158,7 +158,7 @@ type GitRepo struct {
 // Adapts a Secret into a VolumeSource
 type SecretSource struct {
 	// Reference to a Secret
-	Target ObjectReference `json:"target"`
+	Target ObjectReference `json:"target" description:"target is a reference to a secret"`
```

Comment:
Thanks. We'll merge the PR to fix the Travis check right after we merge this PR so more of these don't get merged.